spec/support/fixtures/doc-auth-drop-offs-report.json

```diff
@@ -0,0 +1,46 @@
+10-10-2019 <= date user starts doc auth < 10-24-2019
```

Comment:
The extension to this file is misleading? This is definitely not JSON

Reply:
Good catch